jujugui/static/gui/src/test/index.html

```diff
@@ -71,16 +71,41 @@
   <!-- Load up YUI base, app modules, and test utils -->
   <!-- Since only the tests depend on these files and the prod tests disable
        the YUI loader, we have to include them manually here. -->
-  <script src="/juju-ui/assets/all-yui.js"></script>
-  <script src="/juju-ui/assets/modules.js"></script>
-  <script src="/juju-ui/assets/event-simulate.js"></script>
-  <script src="/juju-ui/assets/node-event-simulate.js"></script>
-  <script src="/juju-ui/assets/javascripts/spin.min.js"></script>
+  <script src="/combo?app/assets/javascripts/yui/yui/yui.js&app/assets/javascripts/yui/loader/loader.js"></script>
+  <script src="/combo?modules.js"></script>
+  <script>
+    var GlobalConfig = {
+      combine: true,
+      base: '/combo?/app/assets/javascripts/yui/',
+      comboBase: '/combo?',
+      maxURLLenght: 1300,
+      root: 'app/assets/javascripts/yui/',
+      groups: {
+        app: {
+            //combine: true,
```

Comment:
ahh yeah leftover, I'll fix - needed it not combined to make debugging possible.